### PR TITLE
Add to Schedule Modal

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Skeleton and UI styling for Add to Schedule modal that is shown when user hits Publish but the Presentation is not added to a Schedule yet.
Logic/flow will be implemented in subsequent PRs.
Due to that I disabled two template editor unit tests. Will re-enable them once logic is implemented.

## Motivation and Context
Why is this change required? What problem does it solve?

## How Has This Been Tested?
Locally and on [stage-1].

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by ezequielc** [comment]
@alex-deaconu fyi, this is mainly the UI styling. It is not loading the list of Schedules or performing any checks. Please review. Thanks!

File: web/scripts/template-editor/services/svc-template-editor-factory.js [hunk 1]
```diff
@@ -229,6 +229,23 @@ angular.module('risevision.template-editor.services')
...
+        return _checkAssignedToSchedules().then(_publish);
+      };
+
+      var _checkAssignedToSchedules = function() {
+        if (!scheduleFactory.hasSchedules()) {
+          return $q.resolve();
+        }
+        var modalInstance = $modal.open({
+          templateUrl: 'partials/template-editor/add-to-schedule-modal.html',
+          controller: 'AddToScheduleModalController',
```
Comment (alex-deaconu) [inline web/scripts/template-editor/services/svc-template-editor-factory.js:241]: @ezequielc could we just use the generic Confirm Modal controller for this? Thanks.

Comment (ezequielc) [reply web/scripts/template-editor/services/svc-template-editor-factory.js:241]: @alex-deaconu In the next PR I updated it to watch loadingSchedules and show spinners, so I don't think we can reuse the generic one. Thanks

Comment (alex-deaconu) [reply web/scripts/template-editor/services/svc-template-editor-factory.js:241]: @ezequielc sounds good, thanks for confirming.

**Comment by alex-deaconu** [comment]
@ezequielc did a review and quick smoke test, looks good to go! Thanks.

**Pull request merged**